web_services/foxx/brightid/index.js

```diff
@@ -169,8 +174,21 @@ const handlers = {
       res.throw(403, 'user is not sponsored', {errorNum: NOT_SPONSORED});
     }
 
-    if (! db.userVerifications(user).includes(context.verification)) {
-      res.throw(404, 'user can not be verified for this context', {errorNum: CAN_NOT_BE_VERIFIED});
+    const verifications = db.userVerifications(user);
+    const all_verifications = db.allVerifications();
```

Comment:
It is very heavy query to iterate over all verifications on every call. `allVerifications` function can be removed and we can hard-code valid verfication names.
Also using `_` in `all_verifications` is not consistent with naming pattern we use here.

Reply:
it's done
https://github.com/BrightID/BrightID-Node/commit/efc1c17ed3db41b5a793067d9b033122ff103af3